.travis.yml

```diff
@@ -268,6 +269,9 @@ matrix:
             - pypy
 
 before_install:
+  - if [[ "${TEST_COVERAGE}" == "true" ]]; then
+      pip install codecov coverage;
```

Comment:
Can you add a line `coverage --version` here? It seems that at least version 3.4 is required. When trying that on my computer I get 4.1. Maybe pip version plays a role here.